src/Webserver/MinkConfiguration.php

```diff
@@ -20,12 +20,17 @@
      * @var
      */
     private $docroot;
+    /**
+     * @var
+     */
+    private $router;
 
-    public function __construct($host, $port, $docroot, $baseUrl)
+    public function __construct($host, $port, $docroot, $router, $baseUrl)
```

Comment:
Erm sure however the config parameter is already optional by definition, if not provided it defaults to `index.php`. Shall I proceed or leave it as it is then?